documentation/UserGuide.md

```diff
@@ -16,6 +17,40 @@ For specific cases, a specialized logger can be used.
     + [Record a single line log](#record-a-single-line-log)
     + [Recording the execution of a task](#recording-the-execution-of-a-task)
   * [Use another logger than the global logger](#use-another-logger-than-the-global-logger)
+  
+## Quick start
+
+This section will explain in 30 seconds how to create a logger for an application and use it. Then we will explore more advance features.
+
+Here is an example fo snippet to create a logger using a file `Progress.log`:
+
+```Smalltalk
+TinyLogger default 
+    addFileLoggerNamed: 'Progress.log'.
+```
+
+This code need to be executed **one time** in the image. For an application, it will probably be added to a method *initialize* on the class side of a class to be executed one time at the loading of the project in the image.
+
+In case you need to execute the code multiple times you can use this snippet instead:
```

Comment:
I'm still confused by why `addFileLoggerNamed: 'x'` would ever make sense to be called multiple times for the same 'x' (or for Transcript).

Also, does `ensureX` remove the log file?

Reply:
Otherwise, perhaps you can give several examples with a specific use-case.

1. Create a log that exists and records all messages as soon as the project is loaded in a a Pharo image. In the *initialize* method (class side) of a project put the following ...
2. Create a log that is reset each time you run an application. Put the following in the method that is run ...

Reply:
#ensureX will not erase the file.

I don't think it should because, for example, if a user launches an application and get info to be logged, then the application crashes and the user launch it again. If you use #ensureX at startup, you don't want to erase the previous logs. 

To reset the logs you should call explicitly #clearLogger (that I will rename #clearLog with your advice).

I did what you said in the Quick Start and separated both scenarios better.